root/admin/edit_user.tt

```diff
@@ -9,13 +9,15 @@
         [% form_row_checkbox(r, 'link_editor', l('Relationship editor')) %]
         [% form_row_checkbox(r, 'location_editor', l('Location editor')) %]
         [% form_row_checkbox(r, 'banner_editor', l('Banner message editor')) %]
+
+        <h2>[%- l('User sanctions') -%]</h2>
         [% form_row_checkbox(r, 'editing_disabled', l('Editing/voting disabled')) %]
         [% form_row_checkbox(r, 'adding_notes_disabled', l('Edit notes disabled')) %]
         [% form_row_checkbox(r, 'spammer', l('Spammer')) %]
```

Comment:
Maybe put Spammer either at top or bottom? (Also, I had been meaning to make a ticket specifically for this, I just hadn’t gotten around to it yet. Ah well. :joy:)

Reply:
You can still make a ticket and I'll link it from the commit message ;)

Reply:
https://tickets.metabrainz.org/browse/MBS-12128